packages/roosterjs-editor-plugins/lib/plugins/ContentEdit/features/listFeatures.ts

```diff
@@ -92,6 +94,30 @@ const OutdentWhenBackOn1stEmptyLine: BuildInEditFeature<PluginKeyboardEvent> = {
     handleEvent: toggleListAndPreventDefault,
 };
 
+/**
+ * MaintainListChainWhenDelete edit feature, provides the ability to indent the list if user press
+ * DELETE before the first item of a list
+ */
+const MaintainListChainWhenDelete: BuildInEditFeature<PluginKeyboardEvent> = {
+    keys: [Keys.DELETE],
+    shouldHandleEvent: (event, editor) => {
+        const li = editor.getElementAtCursor('LI', null /*startFrom*/, event);
+        if (li) {
+            return false;
+        }
+        const nextSibiling = getCacheNextSibiling(event, editor);
+        const nextSibilingIsLI = findClosestElementAncestor(nextSibiling, null, 'li');
```

Comment:
Consider use editor.getElementAtCursor() instead, it wraps findClosestElementAncestor(), and limit the search scope within editor

Reply:
If nextSibiling is null, no need to do further action since we know there isn't LI after current position